hepcrawl/parsers/jats.py

```diff
@@ -513,7 +513,10 @@ def get_reference(self, ref_node):
         )
 
         fields = [
-            ('./source/text()', builder.set_journal_title),
+            ('self::node()[@publication-type="journal" '
```

Comment:
I think you can leave off the `self::node()`, not entirely sure though.

Reply:
I tried and it didn't work. Neither did `.[@attr="val"]`, so this was the best I came up with.